meerkat_libs/db_adapters/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .dynamodb import DynamoDBAdapter
+from .postgresql import PostgreSQLAdapter
```

Comment:
This file makes import statements simpler.

Reply:
I had some troubles in the past with relative imports. Could you use absolute ones instead?
sth like `from meerkat_auth.dynamodb import DynamoDBAdapter` etc.